tests/test_generic.py

```diff
@@ -883,8 +883,14 @@ def test_annotation_builder_highlight(pdf_file_path):
                     FloatObject(705.4493),
                 ]
             ),
+            printing=True,
```

Comment:
We should cover both cases to keep the coverage high (see the lines only being covered partially in the Codecov report).